src/main/java/frc/robot/subsystems/DriveTrainSubSystem.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.util.sendable.SendableRegistry;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.DriveConstants;
+
+public class DriveTrainSubSystem extends SubsystemBase {
+  private final VictorSPX m_leftMotor = new VictorSPX(DriveConstants.kLeftMotorPort);
+  private final VictorSPX m_leftMotor2 = new VictorSPX(DriveConstants.kLeftMotorPort2);
+  private final VictorSPX m_rightMotor = new VictorSPX(DriveConstants.kRightMotorPort);
+  private final VictorSPX m_rightMotor2 = new VictorSPX(DriveConstants.kRightMotorPort2);
+  private DifferentialDrive m_robotDrive;
+
+  public DriveTrainSubSystem() {
+    m_leftMotor2.follow(m_leftMotor);
+    m_rightMotor2.follow(m_rightMotor);
+
+    SendableRegistry.addLW(this, "DriveTrainSubSystem");
+
+    // We need to invert one side of the drivetrain so that positive voltages
+    // result in both sides moving forward. Depending on how your robot's
+    // gearbox is constructed, you might have to invert the left side instead.
+    m_rightMotor.setInverted(true);
+    m_rightMotor2.setInverted(true);
```

Comment:
Do we need to have rightMotor2 inverted since it's already following rightMotor?